java/code/src/com/suse/manager/webui/services/subscriptionmatching/SubscriptionMatchProcessor.java

```diff
@@ -33,19 +33,19 @@
 import com.suse.matcher.json.SystemJson;
 
 import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.tuple.Pair;
 
 import java.util.Collections;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
 
 /**
  * Processes data from the matcher to a form that's displayable by the UI.
- * todo consider caching immediate lookup values to maps to improve performance
+ * consider caching immediate lookup values to maps to improve performance
```

Comment:
Why the todo was removed like this? I don't think the suggested performance improvement has been implemented, hasn't it?

Anyway, since everything related to subscription matching is basically on life support, we can remove the whole line I guess.